modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -106,6 +106,128 @@ definition {
 		}
 	}
 
+	@description = "LPS-173537 - Verify creating an Account entry triggers an action to add a second Account entry"
+	@priority = 5
+	test CanAddAccountEntryAfterCreatingAccountEntry {
+		property portal.acceptance = "true";
+
+		task ("Given a Account object and given an action to add a new Account entry after creating an Account entry") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 180195",
+				objectName = "CustomObject180195",
+				pluralLabelName = "Custom Objects 180195");
+
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "Integer",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "Integer",
+				isRequired = "false",
+				objectName = "CustomObject180195");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject180195");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 180195");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Custom Action",
+				actionName = "CustomAction",
+				active = "true",
+				inputMethod = "true",
+				inputValue = "Object entry added,Business",
+				objectLabel = "Account",
+				thenAction = "Add an Object Entry",
+				whenAction = "On After Add");
+		}
+
+		task ("When the user creates an Account entry") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customField",
+				objectName = "CustomObject180195",
+				value = 0);
+		}
+
+		task ("Then the action is triggered and a second Account entry is added") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 180195");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.assertObjectActions(
+				actionActive = "Yes",
+				actionLabel = "Custom Action",
+				status = "Success");
+		}
+	}
+
+	@description = "LPS-173537 - Verify creating a custom object entry triggers an action to update an Account entry"
```

Comment:
I made a mistake, the description should be:

`Verify updating a custom object entry triggers an action to add an Account entry`